src/lib/transition_router/transition_router.ml

```diff
@@ -176,7 +176,7 @@ let load_frontier ~logger ~verifier ~persistent_frontier ~persistent_root
   | Error (`Failure e) ->
       failwith ("failed to initialize transition frontier: " ^ e)
 
-let wait_for_high_connectivity ~logger ~network =
+let wait_for_high_connectivity ~logger ~network ~is_seed =
```

Comment:
Should we not wait at all, if we're seed

Reply:
Probably, except in the case where we are both a seed and a block producer. We could make this change and be fine for our current deployment workflow, but we would want to add additional checks or logic here to help prevent issues when running custom networks. I will leave it like this for now since this was tested as is and is needed for release; can be convinced to update in develop after the release branch cut.